src/models/Donor.ts

```diff
@@ -0,0 +1,25 @@
+import mongoose, {Schema, Document} from 'mongoose';
+
+export interface DonorDocument extends Document {
+    name: string;
+    longitude: number;
+    latitude: number;
+}
+
+const donorSchema = new Schema<DonorDocument>({
+    name: {
+        type: String,
+        required: true
+    },
+    longitude: {
+        type: String,
```

Comment:
```suggestion
        type: Number,
```
should be consistent with the TypeScript interface